frontend/src/pages/hardwareProfiles/toleration/const.ts

```diff
@@ -37,10 +37,14 @@ export const tolerationColumns: SortableData<Toleration>[] = [
 
 export const EMPTY_TOLERATION: Toleration = {
   key: '',
-  operator: TolerationOperator.EQUAL,
 };
 
 export const operatorDropdownOptions: SimpleSelectOption[] = [
+  {
+    key: '',
```

Comment:
so bc key can be an empty string now, we need to handle this in the onChange for the text input. Operator already does this so follow that example

example
```tsx
 onChange={(key) => {
                const effect = asEnumMember(key, TolerationEffect);
                setToleration('effect', effect ?? undefined);
              }}
```